templates/helm/command-center/templates/argo.yaml

```diff
@@ -54,7 +58,17 @@ spec:
       password:
         secretName: {{ $secretName }}
         secretKey: {{ $keyName }}
-      clusterId: command-center
+    workflowDefaults:
+      {{ $isDev := eq $env "dev" }}
+      podGarbageCollection:
+        enabled: {{ not $isDev }}
+        {{- if not $isDev }}
+        strategy: OnWorkflowSuccess
+        {{- end }}
+      workflowTTL:
+        enabled: true
+        secondsAfterSuccess: {{ if $isDev }}86400{{ else }}3600{{ end }}
+        secondsAfterFailure: {{ if $isDev }}604800{{ else }}259200{{ end }}
```

Comment:
might be helpful to comment with how many days/hours/minutes these seconds correspond to

Reply:
Done!